src/App.js

```diff
@@ -1,7 +1,13 @@
 function App() {
+
+  let array = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse']
+  let listArray = array.map(animal => <li key={animal}>{animal}</li>)
```

Comment:
You can use `const` to declare those variables because they will not change.

Reply:
good idea